docker/azure-services/local-database-version-control-env-template

```diff
@@ -2,6 +2,11 @@
 # against secrets being committed to version control unintentionally.
 
 # Example: postgresql://postgres:<<Replace with password>>@localhost:5432/bng
```

Comment:
As the environment variables are split out now, it could be better to remove this comment and pre-populate non-password values for use with a containerised local environment.